components/generic/CheckBox/CheckBox.tsx

```diff
@@ -0,0 +1,30 @@
+import { ChangeEventHandler } from 'react';
+
+type CheckBoxProps = {
+  name: string;
+  label: string;
+  description?: string;
+  isChecked: boolean;
+  onChange: ChangeEventHandler<HTMLInputElement>;
+};
+
+export const CheckBox = ({ name, label, description, isChecked, onChange }: CheckBoxProps) => {
+  return (
+    <div className="flex flex-col gap-1.5">
+      <label className="flex gap-3">
+        {/* How do we want the checked box to look? */}
```

Comment:
Like on the design.
```suggestion
        {/* How do we want the checked box to look? */}
```

Reply:
I can't seem to find a checked version in the design.

Reply:
You mean unchecked? Top - checked, bottom - unchecked.
![image](https://user-images.githubusercontent.com/47081011/195987435-280dc3f4-1317-4d75-9769-d96e4aa701e4.png)